cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -2014,6 +1926,9 @@ ntel/x86_64/2013/composer_xe_2013/composer_xe_2013_sp1.3.174/mkl/include </base>
   <FFLAGS>
     <append DEBUG="TRUE"> -DSUMMITDEV_PGI -DHAVE_IEEE_ARITHMETIC -DNO_R16 </append>
   </FFLAGS>
+  <LDFLAGS>
+    <append> -Minline -ta=nvidia,cc70,fastmath,loadcache:L1,unroll,fma,managed,ptxinfo -Mcuda -Minfo=accel </append>
+  </LDFLAGS>
```

Comment:
Can `pgigpu` and `pgiacc` compiler entries be merged into one? The two are very similar with `pgigpu` having more recent updates.

Reply:
I'm ok with that, does anyone actually use pgiacc?

Reply:
@mrnorman @jgfouca ^^

Reply:
I doubt that anyone still uses pgiacc configuration now as it was superseded by pgigpu long time ago.

Reply:
ok great, as long as there's nothing in pgiacc I need to keep then I'll just rip it out.

Reply:
There are flags in `pgiacc` that could be useful for CPU portion of runs with MPAS-ocean and MPAS-seaice (in addition to MMF): e.g. https://github.com/E3SM-Project/E3SM/blob/master/cime/config/e3sm/machines/config_compilers.xml#L461. They don't appear to cause merge-conflicts with flags in `pgigpu`.

Reply:
@amametjanov can you specify which flags should be added to pgigpu?

Reply:
Here is the [commit](https://github.com/E3SM-Project/E3SM/commit/72c4eccd387d761a9b88edfb5d9f1eb235ceb0f9) that merges `pgiacc` and `pgigpu`: renames previous `pgiacc` to `pgigpu`, adds custom flags for Summit+`pgigpu`, and removes other occurrences of Summit+`pgiacc`. Would that work?

Reply:
@amametjanov I think that commit will work, I worked it into my branch.
But now I'm seeing this build error when I try to run with pgigpu:

*** Fatal attempt to include C++ code on a compiler/machine combo that has not been set up to support C++.  Stop.

It happens without these changes, so I'm really confused how this got broken. Any ideas what is causing this?

Reply:
bleh, nevermind, I'm an idiot, I deleted the "SUPPORTS_CXX" line.

Reply:
I'm having other build issues now. I think I'm out of my league here. I need someone like @mrnorman to look this over and maybe run a test build with pgigpu.

Reply:
Please try with removal of `OMPI_CXX` in this [commit](https://github.com/E3SM-Project/E3SM/commit/a1a11140da91bc15a361986c36339c252d903c62). I tried
```
./cime/scripts/create_test SMS.f19_g16.X --machine summit --compiler pgigpu --walltime 00:30
```
and it built and ran with no errors. `OMPI_CXX` was pointing mpiCC wrapper to nvcc instead of pgc++ and causing SCORPIO build fail. We can add mpiCC-to-nvcc binding for Kokkos builds in CMake at a later time (tagging @jgfouca ).

Reply:
Looks like that did the trick! thanks